docs/settings.md

```diff
@@ -398,3 +398,8 @@ Or to use manual generated stubs:
 ```
 
 clojure-lsp will generate no stubs with that, but analyze that folder and consider it as manual generated stubs.
+
+
+### Changing settings
+
+`clojure-lsp` supports `workspace/didChangeConfiguration` commands to change the server settings while it is running, everything inside the `:settings` map can be changed, but not outside it (the project root for example).
```

Comment:
maybe worth mentioning that some settings are read only during startup and won't happen nothing if changed after that

Reply:
nice, do we have a list of these settings?